packages/react/src/backlink/Backlink.tsx

```diff
@@ -0,0 +1,50 @@
+import { forwardRef, type Ref } from 'react';
+import { cx } from 'cva';
+import { Link as RACLink, type LinkProps } from 'react-aria-components';
+import { ChevronLeft } from '@obosbbl/grunnmuren-icons-react';
+
+type BacklinkProps = {
+  /** Additional CSS className for the element. */
+  className?: string;
+
+  /** Additional style properties for the element. */
+  style?: React.CSSProperties;
+
+  /** The URL to navigate to when clicking the backlink. */
+  href?: string;
+
+  /** The content of the link */
+  children?: React.ReactNode;
+
+  /** To add a permanent underline on the link (not only on hover)
+   * @default false
+   */
+  withUnderline?: boolean;
```

Comment:
Kom vi ikke til at vi går for å _ikke_ ha prop i første omgang?

Reply:
Jo, men den kom nog in igår hehe

Reply:
My bad, se diskussion över

Reply:
Jo, men så hadde jeg en diskusjon med Kristian der det kom frem en del ting: [https://github.com/code-obos/grunnmuren/pull/763#discussion_r1538998926](https://github.com/code-obos/grunnmuren/pull/763#discussion_r1538998926)